src/index.js

```diff
@@ -779,6 +779,21 @@ export default class ReactTable extends Methods(Lifecycle(Component)) {
         }}
         {...rootProps.rest}
       >
+        {showPagination.top ? (
```

Comment:
I can wrap the top and bottom pagination components with a div in index.js, so then we can only create it once.